src/Lesson_5/Employee.java

```diff
@@ -0,0 +1,33 @@
+package Lesson_5;
+
+public class Employee {
+
+    String name;
+    String surname;
+    String post;
+    String email;
+    String number;
+    double salary;
+    int age;
+
+    public Employee(String name, String surname, String post, String email, String number, double salary, int age) {
+        this.name = name;
+        this.surname = surname;
+        this.post = post;
+        this.email = email;
+        this.number = number;
+        this.salary = salary;
+        this.age = age;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    void showEmp() {
```

Comment:
метод используется вне класса, лучше объявить публичным
```suggestion
    public void showEmp() {
```